tests/telemetry/test_telemetry.py

```diff
@@ -48,9 +48,10 @@ def verify_telemetry_dockerimage(duthost):
     docker_out = duthost.shell('docker images docker-sonic-telemetry', module_ignore_errors=False)['stdout_lines']
     docker_out_list = get_list_stdout(docker_out)
     matching = [s for s in docker_out_list if "docker-sonic-telemetry" in s]
-    if str(matching):
+    if matching:
         return True
-    return False
+    else:
+        return False
```

Comment:
are you trying to do return (len(matching) > 0)?

Reply:
just trying to verify whether the list contains matching elements or not

Reply:
Ying's suggestion is less lines of code.